webapp/src/dogma/common/components/DiffView.tsx

```
@@ -60,12 +66,16 @@ function highlightSyntax(path: string, str: string): React.JSX.Element {
 
-  return (
-    <pre
-      style={{ display: 'inline' }}
-      dangerouslySetInnerHTML={{ __html: Prism.highlight(str, Prism.languages[language], language) }}
-    />
-  );
+  try {
+    return (
+      <pre
+        style={{ display: 'inline' }}
+        dangerouslySetInnerHTML={{ __html: Prism.highlight(str, Prism.languages[language], language) }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)